sql-statements/sql-statement-alter-user.md

```diff
@@ -177,6 +180,23 @@ ALTER USER 'newuser' PASSWORD REUSE INTERVAL 90 DAY;
 Query OK, 0 rows affected (0.02 sec)
 ```
 
+
+通过 `ALTER USER ... WITH MAX_USER_CONNECTIONS N` 修改用户 `newuser` 允许登陆的最大连接数：
+
+```sql
+ALTER USER 'newuser' WITH MAX_USER_CONNECTIONS 3;
+SELECT USER, HOST, MAX_USER_CONNECTIONS FROM MYSQL.USER WHERE USER='newuser';
```

Comment:
```suggestion
SELECT User, Host, max_user_connections FROM mysql.user WHERE User='newuser';
```

The column names are stored with a specific case. While they should work when compared with any other case it is best to keep these identical.

Reply:
Using system tables directly should be avoided.

So insert of this `GRANT`, `CREATE USER`, etc should be used.

In this case could `SHOW CREATE USER...` or `SHOW GRANTS FOR...` be used?

Reply:
I guess this is following the same format as other examples. 

I'm ok if this is changed for all examples in a separate PR some time later.

Reply:
> I guess this is following the same format as other examples.
> 
> I'm ok if this is changed for all examples in a separate PR some time later.

Yes, It is used for showing the field `max_user_connections` when creating user `WITH MAX_CONNECTIONS N`.

Reply:
I think this might be a better way than `SELECT ... mysql.user...`:

```
mysql-8.0.11-TiDB-v9.0.0-alpha-340-gd392685> CREATE USER 'u1'@'%' IDENTIFIED BY 'really_really_secret' WITH MAX_USER_CONNECTIONS 123;
Query OK, 0 rows affected (0.03 sec)

mysql-8.0.11-TiDB-v9.0.0-alpha-340-gd392685> SHOW CREATE USER 'u1'@'%'\G
*************************** 1. row ***************************
CREATE USER for u1@%: CREATE USER 'u1'@'%' IDENTIFIED WITH 'mysql_native_password' AS '*D4EF4C5A994EF8283FB666D79DCADE5B6A764D4E' REQUIRE NONE WITH MAX_USER_CONNECTIONS 123 PASSWORD EXPIRE DEFAULT ACCOUNT UNLOCK PASSWORD HISTORY DEFAULT PASSWORD REUSE INTERVAL DEFAULT
1 row in set (0.01 sec)
```